src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -448,6 +451,51 @@ public void userProfileCreateUserWireMock(HttpStatus status) {
         );
     }
 
+    public void userProfileCreateUserWireMockWithExtraRoles() {
+
+        String body = null;
+        body = "{"
+                + "  \"idamId\":\"" + UUID.randomUUID().toString() + "\","
+                + "  \"idamRegistrationResponse\":\"201\""
+                + "}";
+
+        userProfileService.stubFor(post(urlEqualTo("/v1/userprofile"))
+                .willReturn(aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(body)
+                        .withStatus(201)
+                ));
+
+        String usersBody = "{"
+                + "  \"userProfiles\": ["
+                + "  {"
+                + "  \"userIdentifier\":\"%s" + "\","
+                + "  \"firstName\": \"Adil\","
```

Comment:
Not sure about HMCTS policy, but we usually do not disclose the identity of employees in a public code. Maybe Shashank or Praveen can enlighten this better.
I have seen other dev's names as well.

Reply:
How about James bond??

Reply:
We've had our names (Prashanth, Shreedhar and myself) there for more than a year - can change however as it's not important

Reply:
Let me check and come back to you.

Reply:
I have updated the names - keeping the first name and only the initial for the surname as well as having all emails as `super.user@hmcts.net` which is not a real email